src/core/bpf-socket-bind.c

```diff
@@ -11,7 +11,7 @@
 /* libbpf, clang, llvm and bpftool compile time dependencies are satisfied */
 #include "bpf-dlopen.h"
 #include "bpf-link.h"
-#include "bpf/socket_bind/socket-bind.skel.h"
+#include "bpf/socket_bind/socket-bind-skel.h"
```

Comment:
please stick to .bpf.c and .skel.h conventions, that are used pretty universally across many BPF-based projects

Reply:
The bpftool generated `socket-bind.skel.h` file exists still with the same name as before, it just gets included by `socket-bind-skel.h`(which redefines the libbpf symbols to avoid linker issues). I can't rename it to `socket-bind.skel.h` as the file with that name already exists. Should I rename it to something different?

Reply:
Renamed to `socket-bind-skel-wrapper.h`.